### DISCUSSION TITLE
切换页面不显示

### DISCUSSION THREAD

**Discussion by anjinlenxiao** [zuihou/zuihou-admin-cloud#161] [02.提问]
开发过程中点击左边侧边栏切换时，在已打开几个页面的情况下。页面内容不加载不报错，处于空白情况，在重新刷新之后会显示出来。
![微信图片_20220714153344](https://user-images.githubusercontent.com/85476509/178927509-34ce43cd-182b-4fc1-8583-8e0db768c4a0.png)

**Comment by zuihou** [comment]
无法重现，建议提供重现代码或在官方代码中找出重现页面

**Reply by anjinlenxiao** [reply]
就这个例子来说，我本地运行起来项目后想查看例子，在点击页面权限后报错了，然后我再点击其他的页面都不会加载
![微信图片_20220715150331](https://user-images.githubusercontent.com/85476509/179171230-cfee4f84-ee84-46f4-884d-dab5210de94c.png)

**Comment by zuihou** [comment]
静态示例 中的页面是原封不动的使用了vben的“示例代码"，仅供参考的， 实际项目中最好删除掉。
为了适配lamp后端，修改了一些vben封装的接口或工具类等，导致vben提供的"示例代码"可能会报错。

当前这个页面就是因为修改了 `userStore.getRoleList` 导致 vben提供的页面报错。

**Comment by zuihou** [comment]
但为什么这个页面报错会影响整个框架，还没找到原因
